From 299cb53e44efdbccd2ad623434edcdbd4073547a Mon Sep 17 00:00:00 2001
From: Filipe Fortes <fortes@users.noreply.github.com>
Date: Fri, 17 May 2019 15:02:18 -0400
Subject: [PATCH] Fix ImageMagick Detection

The `return` statement was at the wrong indent level, so we never checked for the legacy `convert` executable
---
 beets/util/artresizer.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/beets/util/artresizer.py b/beets/util/artresizer.py
index 3590dcf22..1ee3e560d 100644
--- a/beets/util/artresizer.py
+++ b/beets/util/artresizer.py
@@ -278,7 +278,7 @@ def get_im_version():
                                int(match.group(3)))
                     return version, legacy
 
-        return None
+    return None
 
 
 def get_pil_version():
